protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -20,15 +22,20 @@
             'form' => $form,
             'model' => $group,
             'attribute' => 'defaultSpaceGuid',
-            'selection' => $group->defaultSpace,
-            'maxSelection' => 1,
+            'selection' => $group->defaultSpaces,
+            'maxSelection' => 1000,
         ])
         ?>
     <?php endif; ?>
 
+    <div class="reassign-spaces">
+        <?= CHtml::a(Yii::t('AdminModule.user', 'Reassign All'), Url::toRoute(['/admin/group/reassign-all', 'id' => $group->id]), ['class' => 'btn btn-primary btn-reassign']); ?>
+        <div class="help-block">Reassign default spaces to all users</div>
```

Comment:
Missing Yii::t - always use the Model `attributeHints` if possible

Reply:
Also `CHtml` is deprecated use `humhub\libs\Html`

Reply:
Done!